src/components/RegistrationFormBody.tsx

```diff
@@ -1,18 +1,19 @@
-import React from 'react';
-import { Form, Input, Radio, Upload, DatePicker } from 'antd';
+import { DatePicker, Form, Input, Radio, Upload } from 'antd';
 import { FormListFieldData } from 'antd/lib/form/FormList';
+import React from 'react';
 const { Dragger } = Upload;
 const { TextArea } = Input;
 
-const RegistrationFormBody: React.FC<{ field: FormListFieldData }> = ({
-  field,
-}) => {
-  const fieldWithoutKey: any = field;
-  delete fieldWithoutKey.key;
+const RegistrationFormBody: React.FC<{
+  field: FormListFieldData;
+  isChildForm: boolean;
+}> = ({ field, isChildForm }) => {
```

Comment:
We should have an interface `RegistrationFormBodyProps` defined above and not exported